dashboard/config/locales/en.yml

```diff
@@ -701,6 +701,7 @@ en:
     required: 'Overview'
     content: 'Content'
     practice: 'Teaching Practices'
+    peer_review: 'Peer Review'
```

Comment:
It will be when I bring in the peer review section. Took it out for now to simplify things